package.json

```diff
@@ -375,7 +376,14 @@
 			"simplemde>codemirror": "^5.65.11",
 			"simplemde>marked": "^4.3.0",
 			"sharp": "^0.33.2",
-			"@fluentui/react-positioning>@floating-ui/dom": "~1.5.4"
+			"@fluentui/react-positioning>@floating-ui/dom": "~1.5.4",
+			"@aws-crypto/sha256-browser": "npm:empty-npm-package@1.0.0",
+			"@aws-crypto/sha256-js": "npm:empty-npm-package@1.0.0",
+			"@aws-sdk/client-cloudfront": "npm:empty-npm-package@1.0.0",
+			"@aws-sdk/client-s3": "npm:empty-npm-package@1.0.0",
+			"@aws-sdk/client-sso-oidc": "npm:empty-npm-package@1.0.0",
+			"@aws-sdk/client-sts": "npm:empty-npm-package@1.0.0",
+			"@aws-sdk/credential-provider-node": "npm:empty-npm-package@1.0.0"
```

Comment:
I think there is an override syntax to target specific packages. It would be great to target oclif explicitly so that any other dependencies aren't inadvertently nulled.

Reply:
Fixed in latest, and I was able to minimize the overrides to two.